src/main/java/mate/jdbc/services/impl/DriverService.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.services.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Driver;
+import mate.jdbc.services.Service;
+
+@mate.jdbc.lib.Service
+public class DriverService implements Service<Driver> {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        return driverDao.get(id);
+    }

```

Comment:
Maybe it is better to return something else but not empty object? But Im not sure about it